lib/bpf/bpf_test.go

```diff
@@ -463,11 +475,11 @@ func executeCommand(c *check.C, doneContext context.Context, file string) {
 }
 
 // executeHTTP will perform a HTTP GET to some endpoint in a loop.
-func executeHTTP(c *check.C, doneContext context.Context, endpoint string) {
+func executeHTTP(t *testing.T, doneContext context.Context, endpoint string) {
 	for {
 		// Perform HTTP GET to the requested endpoint.
 		_, err := http.Get(endpoint)
-		c.Assert(err, check.IsNil)
+		require.NoError(t, err)
```

Comment:
```suggestion
		assert.NoError(t, err)
```

as this and `executeCommand` run on separate goroutines.

`executeCommand` still has a `c.Fatalf` which doesn't compile, I think.